source/guide_roundcube.rst

```diff
@@ -0,0 +1,222 @@
+ Selected PHP version 7.3
+ The new configuration is adapted immediately. Patch updates will be applied automatically.
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.3'
+ [isabell@stardust ~]$
+
+If you want to use your Roundcube Webmail with your own domain you need to setup your domain first:
+
+::
+
+ [isabell@stardust ~]$ uberspace web domain list
+ isabell.uber.space
+ [isabell@stardust ~]$
+ 
+You'll need your MySQL :manual_anchor:`credentials <database-mysql.html#login-credentials>`. Get them with ``my_print_defaults``:
+
+::
+
+ [isabell@stardust ~]$ my_print_defaults client
+ --default-character-set=utf8mb4
+ --user=isabell
+ --password=MySuperSecretPassword
+ [isabell@stardust ~]$
+ 
+
+
+
+Installation
+============
+
+``cd`` to your :manual:`document root <web-documentroot>`, download the latest release of Roundcube and extract it. Move all containing files up one level and remove the emtpty directory as well as the archive afterwards.
+
+.. note:: Check the Roundcube_ website or `Github Repository`_ for the latest stable release and copy the download link to the Complete.tar.gz file. Then use ``wget`` to download it. Replace the URL with the one you just got from GitHub/Website.
+
+.. code-block:: console
+ :emphasize-lines: 2,4,5,6,7,8
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://github.com/roundcube/roundcubemail/releases/download/1.4.3/roundcubemail-1.4.3-complete.tar.gz
+ [isabell@stardust html]$ tar xfz *.tar.gz
+ [isabell@stardust html]$ cd roundcubemail-1.4.3
+ [isabell@stardust roundcubemail-1.4.3]$ mv * .[a-z]* ../ 
+ [isabell@stardust roundcubemail-1.4.3]$ cd ..
+ [isabell@stardust html]$ rm -r roundcubemail-1.4.3/
+ [isabell@stardust html]$ rm -r roundcubemail-1.4.3-complete.tar.gz
+ [isabell@stardust html]$
+ 
+
+Configuration
+=============
+
+Create Database
+---------------
+
+Roundcube saves your data in a :manual:`MySQL <database-mysql>` database. It is recommended to use an :manual_anchor:`additional database <database-mysql.html#additional-databases>` (e.g. ``isabell_roundcube``) instead of the default database.
+
+.. note:: You need to create the database **before** you enter the database :manual_anchor:`credentials <database-mysql.html#login-credentials>` in the `Roundcube Installer`_.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mysql -e "CREATE  DATABASE ${USER}_roundcube"
+ [isabell@stardust ~]$
+
+
+Roundcube Installer
+-------------------
+
+The final configuration can easily be done in the browser.  Point your Browser to your installation URL ``https://isabell.uber.space/installer/``.
+
+All prerequisite checks should be *OK* - click *NEXT*.
+
+.. note:: You can safely ignore the *NOT AVAILABLE*-Warnings at the SQLite, SQL Server etc Databases, as we'll use the MySQL Database.
+
+Enter the following details at the Tab *CREATE CONFIG*:
+
+General configuration
+---------------------
```

Comment:
Please use `^^^^^^^^^^^^^` headings for this and the other configuration inputs. That way they are correctly grouped as part of "Roundcube Installer" instead of being at the same level.

Reply:
changed the format as suggested.